r1cs/gadgets/std/src/to_field_gadget_vec.rs

```diff
@@ -56,27 +57,27 @@ impl<ConstraintF: PrimeField> ToConstraintFieldGadget<ConstraintF> for () {
     }
 }
 
-impl<M, ConstraintF, FG> ToConstraintFieldGadget<ConstraintF>
-    for SWPAffineGadget<M, ConstraintF, FG>
-where
-    M: SWModelParameters,
-    ConstraintF: PrimeField,
-    FG: FieldGadget<M::BaseField, ConstraintF>
-        + ToConstraintFieldGadget<ConstraintF, FieldGadget = FpGadget<ConstraintF>>,
-{
-    type FieldGadget = FpGadget<ConstraintF>;
-
-    #[inline]
-    fn to_field_gadget_elements<CS: ConstraintSystem<ConstraintF>>(
-        &self,
-        mut cs: CS,
-    ) -> Result<Vec<Self::FieldGadget>, Error> {
-        let mut x_fe = self.x.to_field_gadget_elements(cs.ns(|| "x"))?;
-        let y_fe = self.y.to_field_gadget_elements(cs.ns(|| "y"))?;
-        x_fe.extend_from_slice(&y_fe);
-        Ok(x_fe)
-    }
-}
+// impl<M, ConstraintF, FG> ToConstraintFieldGadget<ConstraintF>
```

Comment:
Why this is commented ?

Reply:
Leftover from debugging, I've uncommented this